mne/time_frequency/tests/test_tfr.py

```diff
@@ -55,19 +55,29 @@ def test_time_frequency():
     times = epochs.times
     nave = len(data)
 
+    epochs_picks = Epochs(raw, events, event_id, tmin, tmax,
```

Comment:
I think I used `epochs_picks` because these are the epochs used for the`tfr_morlet` but you're right, it makes sense to call it nopicks since the previous actually picks channels